src/main/resources/config/application.properties

```diff
@@ -30,7 +30,7 @@ subscriptions.mac.secret=${SUBSCRIPTIONS_MAC_SECRET:our-big-secret}
 
 spring.application.name=TribunalsCaseApi
 
-feature.issue_further_evidence=${ISSUE_FURTHER_EVIDENCE_ENABLED:false}
+feature.issue_further_evidence=${ISSUE_FURTHER_EVIDENCE_ENABLED:true}
```

Comment:
Should this be true? This was not meant to be switched on until end of August

Reply:
It's off in demo, aat and prod. They can be switched on at that point in those environments.